pkg/eval/json_evaluator.go

```diff
@@ -274,8 +268,37 @@ func (je *JSONEvaluator) evaluateVariant(
 	return je.state.Flags[flagKey].DefaultVariant, reason, nil
 }
 
+// configToFlags convert string configurations to flags and store them to pointer newFlags
+func (je *JSONEvaluator) configToFlags(config string, newFlags *Flags) error {
+	schemaLoader := gojsonschema.NewStringLoader(schema.FlagdDefinitions)
+	flagStringLoader := gojsonschema.NewStringLoader(config)
+	result, err := gojsonschema.Validate(schemaLoader, flagStringLoader)
+
```

Comment:
err check ought to come directly after the assignment (no return)

Reply:
please check the latest changes :)